coda/coda_mdstore/tests/test_views.py

```diff
@@ -591,22 +591,42 @@ def test_raises_http404_file_handle_is_falsy(self, rf):
         with pytest.raises(http.Http404):
             views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_output_text(self):
-        assert 0
-
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_top_files_block(self):
-        # TODO: rename once implemented.
-        assert 0
-
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_path_is_not_unicode_safe(self):
-        assert 0
+    def test_response_content(self, rf):
+        """Test the output contains domain and parsed tag attributes from getFileHandle."""
+        self.getFileHandle.return_value.url = 'https://coda/testurl'
```

Comment:
Based on the arguments used in calling `getFileHandle` from `bagURLList` (` handle = getFileHandle(identifier, "manifest-md5.txt")`), I think this would make more sense with a value ending in `manifest-md5.txt` as should be the case in practice. Something like `http://coda/bag/123/manifest-md5.txt`.

Reply:
Maybe for a clearer idea of what is happening in the function under test it should be a side effect and return `['http://coda/bag/123/manifest-md5.txt', 'http://coda/bag/123/']`